pandas/core/window/ewm.py

```diff
@@ -219,14 +231,16 @@ def __init__(
         ignore_na: bool = False,
         axis: int = 0,
         times: Optional[Union[str, np.ndarray, FrameOrSeries]] = None,
+        **kwargs,
```

Comment:
what is this added for?

Reply:
This is so `ExpandingMovingWindow` can accept the `groupby` object when a shallow copy is done on the object.

`Rolling` and `Expanding` have similar need for **kwargs in the __init__.